UI/Web/src/app/manga-reader/_components/infinite-scroller/infinite-scroller.component.html

```diff
@@ -13,7 +13,7 @@
     <strong>Scroll Top:</strong> {{getScrollTop()}}
   </div>
 
-  <div *ngIf="atTop" #topSpacer class="spacer top" role="alert" (click)="loadPrevChapter.emit()">
+  <div *ngIf="atTop || heightLessThanView()" #topSpacer [style.height]="heightLessThanView() ? 'inherit' : '300px'" class="spacer top" role="alert" (click)="loadPrevChapter.emit()">
```

Comment:
Help me understand the arbitrary 300px? Is this just from the styles? If so, let's make it a css variable and use var(--) on the property so I don't forget to update it in multiple places.